Modules/signalmodule.c

```diff
@@ -100,14 +100,15 @@ static volatile struct {
 
 static volatile struct {
     SOCKET_T fd;
+    int warn_on_full_buffer;
     int use_send;
-    int send_err_set;
-    int send_errno;
-    int send_win_error;
-} wakeup = {INVALID_FD, 0, 0};
+} wakeup = {INVALID_FD, 1, 0};
```

Comment:
I suggest to use C99 initializers:

wakeup = {.fd = INVALID_FD, .warn_on_full_buffer = 1, use_end=0};

Reply:
Done.